### PR TITLE
chore: icon story moved up to core + docs updated 

### PR DESCRIPTION
**Pull request by dreamwasp**
## Overview
Icon story moved up to core + docs updated + .vscode settings added to auto-enable wordwrap on .mdx files.

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
+  "[mdx]": {
+    "editor.wordWrap": "wordWrapColumn",
+    "editor.wordWrapColumn": 120
+  },
```
Comment (JoshuaKGoldberg) [inline .vscode/settings.json:5]: Err, unrelated?

**Review by JoshuaKGoldberg — APPROVED** [review]
Yes!

**Review by codecaaron — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.10.4-alpha.9cedee.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fa99d2663a5f9014253b246--gamut-preview.netlify.app>https://5fa99d2663a5f9014253b246--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fa99d2663a5f9014253b246>Deploy Logs</a></p>

File: .vscode/settings.json (hunk 1 above)
Comment (dreamwasp) [reply .vscode/settings.json:5]: paired w/ @codecaaron + said it would be helpful to have these settings in the codebase. added to PR description too!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@18.1.1-alpha.152a08.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fa9a17d376c2700bcc8d44d--gamut-preview.netlify.app>https://5fa9a17d376c2700bcc8d44d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fa9a17d376c2700bcc8d44d>Deploy Logs</a></p>

**Pull request merged**